float64 shoulder_Y
float64 shoulder_X
float64 elbow_Z
float64 elbow_X
float64 wrist_Z
float64 wrist_X
float64 thumb
float64 finger_thumb
float64 phalanx_thumb_1
float64 phalanx_thumb_2
float64 phalanx_thumb_3
float64 lit_finger
float64 phalanx_lit_finger_1
float64 phalanx_lit_finger_2
float64 phalanx_lit_finger_3
float64 ring_finger
float64 phalanx_ring_finger_1
float64 phalanx_ring_finger_2
float64 phalanx_ring_finger_3
float64 mid_finger
float64 phalanx_mid_finger_1
float64 phalanx_mid_finger_2
float64 phalanx_mid_finger_3
float64 forefinger
float64 phalanx_forefinger_1
float64 phalanx_forefinger_2
float64 phalanx_forefinger_3
---
float64 shoulder_Y
float64 shoulder_X
float64 elbow_Z
float64 elbow_X
float64 wrist_Z
float64 wrist_X
float64 thumb
float64 finger_thumb
float64 phalanx_thumb_1
float64 phalanx_thumb_2
float64 phalanx_thumb_3
float64 lit_finger
float64 phalanx_lit_finger_1
float64 phalanx_lit_finger_2
float64 phalanx_lit_finger_3
float64 ring_finger
float64 phalanx_ring_finger_1
float64 phalanx_ring_finger_2
float64 phalanx_ring_finger_3
float64 mid_finger
float64 phalanx_mid_finger_1
float64 phalanx_mid_finger_2
float64 phalanx_mid_finger_3
float64 forefinger
float64 phalanx_forefinger_1
float64 phalanx_forefinger_2
float64 phalanx_forefinger_3